qemu/tests/cfg/nvdimm.cfg

```diff
@@ -36,3 +36,12 @@
             mem-path = ${nv_backend}
             del pre_command
             del post_command
+        - nvdimm_dax:
+            start_vm = "no"
+            nvdimm_dax = "yes"
+            nv_backend = "/dev/dax0.0"
+            mem-path = ${nv_backend}
+            del pre_command
+            del post_command
+            ndctl_install_cmd = "rpm -q ndctl || yum install -y ndctl"
```

Comment:
no need to do "rpm -q ndctl".    "yum install -y ndctl"  will skip install and return "True"  if package exist.

Reply:
Modified, Thanks!